BEIMA.Backend/DeviceFunctions/AddDevice.cs

```diff
@@ -34,6 +35,14 @@ public static class AddDevice
             log.LogInformation("C# HTTP trigger function processed a device post request.");
 
 
+            var authService = AuthenticationDefinition.AuthticationInstance;
```

Comment:
```suggestion
            var authService = AuthenticationDefinition.AuthenticationInstance;
```
There is a spelling mistake on the class property, we should probs fix that.